pkg/freeze/containerd/pause.go

```diff
@@ -89,27 +92,31 @@ func lookupContainerID(ctx context.Context, conn *grpc.ClientConn, podUID, conta
 		},
 	})
 	if err != nil {
-		return "", err
+		return []string{}, err
 	}
 
 	if len(pods.Items) == 0 {
-		return "", fmt.Errorf("pod %s not found", podUID)
+		return []string{}, fmt.Errorf("pod %s not found", podUID)
 	}
 	pod := pods.Items[0]
 
 	ctrs, err := client.ListContainers(ctx, &cri.ListContainersRequest{Filter: &cri.ContainerFilter{
 		PodSandboxId: pod.Id,
-		LabelSelector: map[string]string{
-			"io.kubernetes.container.name": containerName,
-		},
 	}})
 	if err != nil {
-		return "", err
+		return []string{}, err
+	}
+
+	var ids []string
+	for _, c := range ctrs.Containers {
```

Comment:
probably worth adding some extra unit tests for this

Reply:
Will do, just need to learn a bit about the various FakeRuntimeService et al. helpers to get those set up